include/splinterdb/splinterdb.h

```diff
@@ -97,9 +97,14 @@ splinterdb_open(const splinterdb_config *cfg, splinterdb **kvs);
 
 // Close a splinterdb
 //
-// This will flush all data to disk and release all resources
+// This will flush all data to disk and release all resources.
+// Handle is NULL'ed out on a succcessful close.
 void
-splinterdb_close(splinterdb *kvs);
+splinterdb_close(splinterdb **kvs);
+
+// Check status of a splinterdb instance: Is it running?
+bool
+splinterdb_is_up(const splinterdb *kvs);
```

Comment:
This is a convenience testing and probe-hook for applications to call. I am bringing this in as part of the examples, but we should probably add this on its commit (for better tracking).

Reply:
(Have reverted these diffs, so that this change-set reduces to just the C-sample programs. Ignore this diff.)